src/library/Uno.Material/Styles/Controls/v2/FloatingActionButton.xaml

```diff
@@ -1,88 +1,160 @@
+
+			<StaticResource x:Key="FabStateOverlayFocused" ResourceKey="OnPrimaryContainerFocusedBrush" />
+
+			<StaticResource x:Key="FabStateOverlayPressed" ResourceKey="OnPrimaryContainerPressedBrush" />
+
+			<StaticResource x:Key="FabStateOverlayDisabled" ResourceKey="OnSurfaceDisabledLowBrush" />
+			<!--#endregion-->
+
+			<!--#region Surface FAB Brushes-->
+			<StaticResource x:Key="FabSurfaceForeground" ResourceKey="PrimaryBrush" />
+
+			<StaticResource x:Key="FabSurfaceBackground" ResourceKey="SurfaceBrush" />
+
+			<StaticResource x:Key="FabSurfaceForegroundDisabled" ResourceKey="OnSurfaceDisabledBrush" />
+
+			<StaticResource x:Key="FabSurfaceStateOverlayPointerOver" ResourceKey="PrimaryHoverBrush" />
+
+			<StaticResource x:Key="FabSurfaceStateOverlayFocused" ResourceKey="PrimaryFocusedBrush" />
+
+			<StaticResource x:Key="FabSurfaceStateOverlayPressed" ResourceKey="PrimaryPressedBrush" />
+
+			<StaticResource x:Key="FabSurfaceBackgroundDisabled" ResourceKey="SystemControlTransparentBrush" />
+
+			<StaticResource x:Key="FabSurfaceStateOverlayDisabled" ResourceKey="OnSurfaceDisabledLowBrush" />
+			<!--#endregion-->
+
+			<!--#region Secondary FAB Brushes-->
+			<StaticResource x:Key="FabSecondaryForeground" ResourceKey="OnSecondaryContainerBrush" />
+
+			<StaticResource x:Key="FabSecondaryBackground" ResourceKey="SecondaryContainerBrush" />
+
+			<StaticResource x:Key="FabSecondaryBackgroundDisabled" ResourceKey="SystemControlTransparentBrush" />
+
+			<StaticResource x:Key="FabSecondaryForegroundDisabled" ResourceKey="OnSurfaceDisabledBrush" />
+
+			<StaticResource x:Key="FabSecondaryStateOverlayPointerOver" ResourceKey="OnSecondaryContainerHoverBrush" />
+
+			<StaticResource x:Key="FabSecondaryStateOverlayFocused" ResourceKey="OnSecondaryContainerFocusedBrush" />
+
+			<StaticResource x:Key="FabSecondaryStateOverlayPressed" ResourceKey="OnSecondaryContainerPressedBrush" />
+
+			<StaticResource x:Key="FabSecondaryStateOverlayDisabled" ResourceKey="OnSurfaceDisabledLowBrush" />
+			<!--#endregion-->
+
+			<!--#region Tertiary FAB Brushes-->
+			<StaticResource x:Key="FabTertiaryForeground" ResourceKey="OnTertiaryContainerBrush" />
+
+			<StaticResource x:Key="FabTertiaryBackground" ResourceKey="TertiaryContainerBrush" />
+
+			<StaticResource x:Key="FabTertiaryBackgroundDisabled" ResourceKey="SystemControlTransparentBrush" />
+
+			<StaticResource x:Key="FabTertiaryForegroundDisabled" ResourceKey="OnSurfaceDisabledBrush" />
+
+			<StaticResource x:Key="FabTertiaryStateOverlayPointerOver" ResourceKey="OnTertiaryContainerHoverBrush" />
+
+			<StaticResource x:Key="FabTertiaryStateOverlayFocused" ResourceKey="OnTertiaryContainerFocusedBrush" />
+
+			<StaticResource x:Key="FabTertiaryStateOverlayPressed" ResourceKey="OnTertiaryContainerFocusedBrush" />
+
+			<StaticResource x:Key="FabTertiaryStateOverlayDisabled" ResourceKey="OnSurfaceDisabledLowBrush" />
```

Comment:
We can remove the blank lines between StaticResources within each `#region` grouping